src/BatteryStateDisplayService.cpp

```diff
@@ -51,7 +51,8 @@ BatteryStateDisplayService::~BatteryStateDisplayService()
 void BatteryStateDisplayService::handleBatteryDataReceived(const BatteryData& batteryData)
 {
     batteryStateOfChargeService_.addData(batteryData);
-
+    batteryStateOfChargeService_.isCharging();
+    qDebug()<<"Time finish charging: " <<batteryStateOfChargeService_.timeWhenChargedOrDepleted();
     QTextStream(stdout) << "Voltage: " << batteryData.voltage
```

Comment:
Since this section of code here is already using `QTextStream` you should follow suit and use the same for debugging rather than using `QDebug`.